src/fs/reader.ts

```diff
@@ -1,26 +1,27 @@
-import { IParserResult } from '@stoplight/types';
-import { parseWithPointers } from '@stoplight/yaml';
-import { existsSync, readFileSync } from 'fs';
-// @ts-ignore
-import * as fetch from 'node-fetch';
+import { readFile } from 'fs';
+const fetch = require('node-fetch');
+import { promisify } from 'util';
+
+const readFileAsync = promisify(readFile);
+
+export const isURL = (uri: string) => /^https?:\/\//.test(uri);
```

Comment:
We could use `parse`, but this would involve parsing of the whole uri, which is unlikely to be needed here. 
I could move that function over to stoplight/path, though.
@wmhilton thoughts?

Reply:
We've got an `isURL` function in `@stoplight/path` 🙂